test/helpers.py

```diff
@@ -172,3 +172,9 @@ def waitport(port, prog, perTickCheck=None):
             else:
                 raise
     raise(Exception("timed out waiting for debug port %d (%s)" % (port, prog)))
+
+def waithealth(prog, addr):
+    subprocess.check_call([
+        'bin/health-checker',
```

Comment:
Nit: elsewhere when we run binaries relative to the current directory we prefix them with `./`. I'd like to stay consistent.